code/__defines/skills.dm

```diff
@@ -1,9 +1,9 @@
-#define SKILL_NONE     1
+#define SKILL_UNSKILLED     1
 #define SKILL_BASIC    2
-#define SKILL_ADEPT    3
-#define SKILL_EXPERT   4
-#define SKILL_PROF     5
-#define HAS_PERK       SKILL_NONE + 1
+#define SKILL_TRAINED    3
+#define SKILL_EXPERIENCED   4
```

Comment:
pls fix spacing

Reply:
Spacing issues also fixed.  Not sure why it didn't mark this as also being outdated?